.zuul.d/playbooks/tox/main.yaml

```diff
@@ -0,0 +1,18 @@
+- hosts: all
+  tasks:
+
+    # TODO(ianw) : let's see if this is interferring
+    # with some of the pip tox runs
+    - name: Remove OpenDev mirror config
+      file:
+        path: /etc/pip.conf
+        state: absent
+      become: yes
```

Comment:
Maybe set `PIP_CONFIG_FILE=/dev/null` env var instead?
cc @pradyunsg

Reply:
If yes, this could probably be done per-test to make use of the mirrors in tests that don't need to hit the production PyPI.

Reply:
Or even `PIP_ISOLATED=true` (is this a thing?)